tests/Command/GenerateAdminCommandTest.php

```diff
@@ -63,6 +64,10 @@ class GenerateAdminCommandTest extends TestCase
 
     protected function setUp(): void
     {
+        if (!class_exists(SensioGeneratorBundle::class)) {
```

Comment:
AFAIK you can remove this class, as this test will never be executed

Reply:
I thought about adding the dependencies to travis to execute the tests, but since they are deprecated, I can just remove them.